### PR TITLE
Enable keyboard shortcuts in calendar

### PR DESCRIPTION
**Pull request by georgegevoian**

### CODE REVIEW

File: calendar/page.js [hunk 1]
```diff
@@ -220,6 +220,8 @@ async function configureGristSettings() {
   // When options changed in the widget configuration (reaction to perspective change)
   grist.onOptions(onGristSettingsChanged);
 
+  grist.enableKeyboardShortcuts();
```
Comment (berhalak) [inline calendar/page.js:223]: Calendar widget loads this file from the `main` source, so this PR will break this widget for a while. So maybe some coordination with release, or some conditional here should be added.

Comment (georgegevoian) [reply calendar/page.js:223]: Any preference? I was thinking of assigning a task for myself to monitor the release Sunday night and merge this after it succeeds.

Comment (paulfitz) [reply calendar/page.js:223]: I'd vote for `enableKeyboardShortcuts?.()` for now so that you don't have to do heroics @georgegevoian.

Comment (JakubSerafin) [reply calendar/page.js]: Maybe conditionally calling this method if it exist with TODO to remove condition in latter term?

**Review by berhalak — APPROVED** [review]
Thanks.

**Pull request merged**